Source/ACE/Entity/Player.cs

```diff
@@ -1605,5 +1605,30 @@ public void ClearDelayedTeleport()
             DelayedTeleportTime = DateTime.MinValue;
             WaitingForDelayedTeleport = false;
         }
+
+        override public void GameLoopUpdate()
+        {
+            uint oldHealth = Health.Current;
```

Comment:
I could do that.  I didn't want to add unneeded data to a class which was used for more things than live updates (perhaps there should be a CreatureVital which extends CreatureAbility?).